lib/migration.js

```diff
@@ -437,28 +364,9 @@ function mixinMigration(Oracle) {
     async.series(tasks, cb);
   };
 
-  /*!
-   * Find the column type for a given model property
-   *
-   * @param {String} model The model name
-   * @param {String} property The property name
-   * @returns {String} The column type
-   */
-  Oracle.prototype.columnDataType = function(model, property) {
-    var columnMetadata = this.columnMetadata(model, property);
-    var colType = columnMetadata && columnMetadata.dataType;
-    if (colType) {
-      colType = colType.toUpperCase();
-    }
-    var prop = this.getModelDefinition(model).properties[property];
-    if (!prop) {
-      return null;
-    }
-    var colLength = columnMetadata && columnMetadata.dataLength || prop.length;
-    if (colType) {
-      return colType + (colLength ? '(' + colLength + ')' : '');
-    }
-
+  Oracle.prototype.buildColumnType = function
+  buildColumnType(propertyDefinition, colLength) {
+    var prop = propertyDefinition;
     switch (prop.type.name) {
```

Comment:
In base connector we do have `columnDataType` and expects the child connectors to implement `buildColumnType`.